README.adoc

```diff
@@ -296,7 +307,7 @@ See the following descriptions of the test cases:
 
 * [hotspot=testUnknownHost file=0]`testUnknownHost()` verifies that an unknown host or a host that does not expose their JVM system properties is correctly handled with a fail message.
 
-=== Running the tests
+=== Running the tests in dev mode
```

Comment:
```suggestion
=== Running the tests in the development environment
```

 To avoid confusion between dev mode and development environment. Although tests are run in dev mode here, the contrast with the next section is between environments development vs test.